MapboxDirections/MBRouteLeg.swift

```diff
@@ -129,10 +134,13 @@ open class RouteLeg: NSObject, NSSecureCoding {
      */
     open let steps: [RouteStep]
     
+    open let segmentDistances: [CLLocationDistance]?
+    
+    open let segmentExpectedTravelTimes: [TimeInterval]?
```

Comment:
The name of this property makes it sound like a verb, “to segment the expected travel times”. Rename this property to `expectedSegmentTravelTimes`.